spec/functions/datetime.md

```diff
@@ -280,11 +283,12 @@ the functions `:datetime`, `:date`, and `:time`.
 - `numberingSystem`
   - valid [Unicode Number System Identifier](https://unicode.org/reports/tr35/tr35.html#UnicodeNumberSystemIdentifier)
 
-The following _option_ and its values are **Proposed** for
-inclusion in the next release of this specification but have not yet been
-finalized.
-If accepted, implementations could be REQUIRED to make this _option_
-available in the functions `:datetime`, `:date`, and `:time`.
+> [!IMPORTANT]
+> The following _option_ and its values are **Draft** and proposed for
+> inclusion in a future release of this specification but have not yet been
+> finalized.
+> If accepted, implementations could be REQUIRED to make this _option_
+> available in the functions `:datetime`, `:date`, and `:time`.
```

Comment:
Everything here is draft, so we no longer need to note this as anything special.

```suggestion
The following _option_ and its values are REQUIRED to be available on
the functions `:datetime`, `:date`, and `:time`.
```